_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5090,20 +5090,104 @@ mfxU64 make_back_color_yuv(mfxU16 bit_depth, mfxU16 Y, mfxU16 U, mfxU16 V)
 };
 
 inline
-mfxU64 make_def_back_color_yuv(mfxU16 bit_depth)
+mfxU64 make_back_color_argb(mfxU16 bit_depth, mfxU16 R, mfxU16 G, mfxU16 B)
 {
-    assert(bit_depth);
+    VM_ASSERT(bit_depth >=8);
+
+    mfxU64 const shift = bit_depth - 8;
+    mfxU64 const max_val = 256 << shift;
```

Comment:
A bit simpler:
mfxU64 const max_val = 1 << bit_depth;
also -1 here is possible